packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -0,0 +1,86 @@
+let INCREMENT = 0n;
+
+/**
+ * A class for parsing snowflake ids
+ */
+export class Snowflake {
+	/**
+	 * The id in BigInt form
+	 */
+	public id: bigint;
+
+	/**
+	 * The timestamp stored in the snowflake
+	 */
+	public timestamp: number;
+
+	/**
+	 * The worker id stored in the snowflake
+	 */
+	public workerID: number;
+
+	/**
+	 * The process id stored in the snowflake
+	 */
+	public processID: number;
+
+	/**
+	 * The increment stored in the snowflake
+	 */
+	public increment: number;
+
+	/**
+	 * The epoch to use in the snowflake
+	 */
+	public epoch: number | bigint;
```

Comment:
make this always a bigint

Reply:
![image](https://user-images.githubusercontent.com/4019718/92289692-99adad00-ef11-11ea-953e-2de7e996fe2b.png)